ui/src/components/Tasks/DailyTasks/index.tsx

```diff
@@ -95,6 +98,15 @@ const DailyTaskRow: React.FC<DailyTask & { isFetching: boolean }> = ({ isFetchin
     setModalOpen(false);
   };
 
+  const dateFromNow = () => {
+    const day = dayjs(p.date);
+    if (!day.isBefore(dayjs(), 'day')) {
```

Comment:
Is this working as expected? Shouldn't this be `if (day.isAfter(dayjs(), 'day')`?

Reply:
it is working with `isBefore`